source/upgrade.txt

```diff
@@ -76,6 +52,14 @@ changes between the current and upgrade versions. For example, if you
 are upgrading the driver from v4.0 to v4.5, address all breaking changes from
 the version after v4.0 including any listed under v4.5.
 
+.. _java-breaking-changes-v4.8:
+
+Version 4.8 Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- The driver ends support for connecting to MongoDB Server versions v3.4 and
```

Comment:
Issue: AFAICT, this is the only place on the page that capitalizes the S in "MongoDB Server."

Suggestion: Standardize capitalization across the page.

Reply:
Good catch. I'll change this instance. And standardizing beyond that such as across the docs set is definitely a worthwhile task, so I'll ticket that.

Reply:
For further clarification, I think "MongoDB Server" is the correct product name, and MongoDB server could be referencing "a server/process that happens to be MongoDB Server". I don't know that our Server Manual uses this consistently, but will do some additional investigation.